gradle/libs.versions.toml

```diff
@@ -100,6 +101,7 @@ avro-avro = { module = "org.apache.avro:avro", version.ref = "avro" }
 awssdk-bom = { module = "software.amazon.awssdk:bom", version.ref = "awssdk-bom" }
 awssdk-s3accessgrants = { module = "software.amazon.s3.accessgrants:aws-s3-accessgrants-java-plugin", version.ref = "awssdk-s3accessgrants" }
 azuresdk-bom = { module = "com.azure:azure-sdk-bom", version.ref = "azuresdk-bom" }
+bson = { module = "org.mongodb:bson", version.ref = "bson-ver"}
```

Comment:
We need to verify this addition is captured in LICENSE/NOTICE files.

Reply:
added section for mongodb bson libs to LICENSE

Reply:
I think we can remove it until we add the transform library to the runtime. When we do that, then the new license plugin can add what is needed.